### ISSUE TITLE
'Publish' button appears in place of 'Update' button

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1954]
**Source**
LF

**Describe the bug**
The 'Publish' button displays after updating a page. This appears to happen randomly.

**To Reproduce**
1. Go to Page admin.
2. Make an edit.
3. Hit 'Update.'
4. 'Update' button changes to 'Publish.'

**Expected behavior**

Button should continue to say 'Update.'

**Screenshots**

<img width="1011" alt="Screen Shot 2022-01-18 at 6 09 15 PM" src="https://user-images.githubusercontent.com/1569463/150896240-a56f46c6-8b9a-4067-a6b2-c429473e6603.png">

**Desktop (please complete the following information):**
 - Mac
 - Firefox

**Additional context**

Seems to be happening randomly.

**Comment by kevindherman** [comment]
@lukefretwell in your screenshot the status says "draft" so you can either save draft or publish. Once you publish (no longer draft) then the publish button should turn to "update" but it appears to be linked to being a draft. I don't know if it's switching to draft unintentionally or on purpose but from the screenshot that seems to be logic.

**Comment by lukefretwell** [comment]
@kevindherman yes, it automatically moves it back to draft after saving, even though it's already published and I'm just updating the page.

**Comment by lukefretwell** [comment]
Appears to be when you add a row, add the code widget, add code, then save. Sometimes it doesn't happen but then when you delete the row, it reverts back to publish.

**Comment by kevindherman** [comment]
Yeah, that's odd.

**Comment by kevindherman** [comment]
@curtismchale this may be isolated to pc.com

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell can someone get me an account on proudcity.com so I can test this there. I can't reproduce it locally.

**Comment by lukefretwell** [comment]
@curtismchale I sent you the invite email with login information.

**Comment by lukefretwell** [comment]
@curtismchale FYI I've been updating the proudcity.com and haven't seen this issue today.

**Comment by curtismchale** [comment]
@lukefretwell did it used to happen every day or every time you edited the site?

**Comment by lukefretwell** [comment]
@curtismchale it was random when I edited the site. I purged the cache from the admin bar and am wondering if that helped fix the issue.
